### PR TITLE
fix swagger docs and make sure docs validation runs

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/api/server/register_images.go [hunk 1]
```diff
@@ -638,25 +638,25 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
...
+	//     type: string
+	//   - in: query
+	//     name: message
+	//     description: Set commit message for imported image
+	//     type: string
+	//   - in: body
+	//     name: request
+	//     description: tarball of container image
+	//     schema:
+	//       type: string
```
Comment (jwhonce) [inline pkg/api/server/register_images.go]: If this is tarball that is not uuencoded, I think you want to add the property of "format: binary".

File: pkg/api/server/register_images.go [hunk 2]
```diff
@@ -665,31 +665,31 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
...
+	//     description: Set commit message for imported image
+	//     type: string
+	//   - in: query
+	//     name: url
+	//     description: Specify a URL instead of a tarball
+	//     type: boolean
+	//   - in: body
+	//     name: request
+	//     schema:
+	//       type: string
```
Comment (jwhonce) [inline pkg/api/server/register_images.go]: Again, I think we want the 'format: binary' property here.

**Comment by baude** [comment]
updated

**Pull request merged**